index.html

```diff
@@ -48,113 +48,60 @@
                         <li><a href="about.html">Join</a></li>
                         <li><a href="about.html">Sponsor</a></li>
                         <li><a href="about.html">News</a></li>
-                        <li><a href="about.html">CC Campaign</a></li>
+                        <li><a href="about.html">Courses</a></li>
                     </ul>
                 </div>
             </div>   
         </nav>
         <div class="headline">
             <h3>"Hello Digital World"</h3>
```

Comment:
[Optional]

- [ ] Kindly fix the header issue. In mobile version it overlaps on itself 👍 

![image](https://user-images.githubusercontent.com/50721479/203387846-ef2fde69-539d-45f3-b122-27a2998a0330.png)